js/NavMainComponent.js

```diff
@@ -53,8 +53,8 @@ NavMainComponent.prototype.init = function () {
         if ($self.text() === 'Menu') {
             $self.text('Close');
             $self.attr('aria-expanded', 'true');
-            component.$brandingLink.attr('tabindex', '3');
-            component.$primaryNavLinks.attr('tabindex', '3');
+            component.$brandingLink.attr('tabindex', '0');
```

Comment:
Why do we need to set a zero tabindex here and below?

Reply:
We switch between tabindex `0` and `-1`, we could remove the tabindex now we're not switching between values greater than 0, however toggling between these `0` and `-1` was a bit simpler to keep track of in code and through tests.